src/remote-config.ts

```diff
@@ -73,6 +73,7 @@ export type RemoteConfig = {
   use_flexible_on_egressMode: boolean;
   use_trygg_overgang_qr_code: boolean;
   vehicles_poll_interval: number;
+  enable_in_app_review: boolean;
```

Comment:
We should try to keep these flags sorted alphabetically, so they're easier to find in the debug screen.